phoenix5-spark/src/it/scala/org/apache/phoenix/spark/PhoenixSparkDatasourceV1ITTenantSpecific.scala

```diff
@@ -105,39 +102,6 @@ class PhoenixSparkITTenantSpecific extends AbstractPhoenixSparkIT {
     verifyResults
   }
 
-  ignore("Can write a DataFrame using 'DataFrame.write' to tenant-specific view - Spark2 sparse columns") {
```

Comment:
I am not sure why this is ignored, but I assume that there is a bug that this would catch.
It'd be better keep this.

Reply:
This class is for datasourceV1 multitenant tests. And this test was a datasource V2 test.
I faced issues with multitenant table. I rewrote this test and all other multitenant tests in the PhoenixSparkITTenantSpecific.